packages/material-ui/src/locale/index.js

```diff
@@ -156,9 +156,9 @@ export const caES = {
 
 export const csCZ = {
   props: {
-    // MuiBreadcrumbs: {
-    //   expandText: 'Show path',
-    // },
+    MuiBreadcrumbs: {
+      expandText: 'Ukázat drobečky',
```

Comment:
I think "drobečky" is frontend designer / UX slang, similar to "breadcrumbs". I wouldn't say users will know what does it mean. The original says "show path", so why not "Ukázat cestu"?

Reply:
Changed, thanks